openhtf/exe/__init__.py

```diff
@@ -51,9 +51,9 @@ def __init__(self, test_data, test_start, teardown_function=None):
     super(TestExecutor, self).__init__(name='TestExecutorThread')
 
     self._teardown_function = (
-        teardown_function and openhtf.PhaseInfo.WrapOrCopy(teardown_function))
-    self._test_data = test_data
-    self._plug_manager = plugs.PlugManager()
+        teardown_function
+        and openhtf.PhaseDescriptor.WrapOrCopy(teardown_function))
+    self._test_desc = test_data
```

Comment:
ah, meant to, thanks.  also renamed to `test_descriptor` because unnecessary abbreviations are annoying.